www/main/overlay.js

```diff
@@ -7,7 +7,7 @@ const { obs: defaultPrefs } = require('./configs/defaults.json');
 
 const { ipcRenderer, remote } = electron;
 
-const { store, i18n } = remote.require('./app');
+const { store, i18n, overlayPresets } = remote.require('./app');
```

Comment:
does this need to come through `remote.require('./app')`? can't you include the json file directly here since it's only used in this window?

Reply:
yeah you are right!

Reply:
Fixed